dart/tool/service_undocumented.md

```diff
@@ -158,3 +158,53 @@ class HeapSpace extends Response {
 <!-- _getCpuProfileTimeline -->
 
 <!-- _getAllocationSamples -->
+
+streamId | event types provided
```

Comment:
Ah, I see why you have to merge above - portions of `EventKind` and `Event` are defined in two separate files. We could merge your change as is, but hopefully the split (some things defined in `service_undocumented.md`) is temporary. I'm wondering if there's not a simpler way to do this; in the parsing code, get `EventKind` and stick two new fields on it; get `Event` and stick 3 new fields on that.

Thoughts?